### DISCUSSION TITLE
Releasing a blog post as a teaser

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [ekqt/screenshot#68] [Ideas]
Hey, guys. I'm thinking about publishing a blog post as a teaser to attract more stars and traction. I've created these slides. Plot twist, they have been created using the app. Let me know what you think, I'll prepare something short and publish it tomorrow.

Of course, I did used the tool and then added the Text in Figma.

![Slide-0](https://github.com/iamhectorsosa/screenshot/assets/77650775/f996a324-a7b6-4a1b-89f2-cfc0915b0dfa)
![Slide-1](https://github.com/iamhectorsosa/screenshot/assets/77650775/ef8d7a0f-ec50-480d-8a57-4830d161f3bb)
![Slide-2](https://github.com/iamhectorsosa/screenshot/assets/77650775/c1b8f311-2be9-4e87-af47-10abb90f3ecc)
![Slide-3](https://github.com/iamhectorsosa/screenshot/assets/77650775/4ea7e0eb-156c-4156-acba-dcc636618326)

@alefDev-prog @kodbilenadam @mateusabelli

**Comment by mateusabelli** [comment]
Hello there, I think that they look very nice. It's awesome the fact that they were created using the tool!
In the future, maybe we could have built in support for text too, what do you think?

**Comment by alefDev-prog** [comment]
I think they look very good, they will surely attract users!
